docs/static/style-guide/formats.md

```diff
@@ -0,0 +1,203 @@
+> This library provides various functions for formatting the data in the preferred style.
+
+## Formatting Functions
+
+#### `sszvis.formatText(x)`
+
+Any given data is returned as a string and will be formatted as Text.
+
+```table
+rows:
+  - Function: formatText
+    Example: sszvis.formatText(16000)
+    Result: "'16000'"
+  - Function:
+    Example: sszvis.formatText("Jahr")
+    Result: "'Jahr'"
+```
+
+#### `sszvis.formatNumber(x)`
+
+A number greater than 9'999 will have a thousands separator space.
+
+```table
+rows:
+  - Function: formatNumber
+    Example: sszvis.formatNumber(16000)
+    Result: "16 000"
+  - Function:
+    Example: sszvis.formatNumber(5000)
+    Result: "5000"
+```
+
+#### `sszvis.formatPreciseNumber(p, d)`
+
+`p` = The desired precision
+`d` = The number to be formatted
```

Comment:
Added these for a more detailed explanation of the function

Reply:
I like these!